packages/ui/src/components/va-toast/hooks/useToast.ts

```diff
@@ -18,6 +19,10 @@ export const useToast = () => {
     return id
   }
 
+  const notify = (options: string | NotificationOptions) => {
```

Comment:
It probably should be another way round.

init should call notify :)

so that when notify is used - no deprecation warnings will happen

and when init is used - deprecation warning would happen

Reply:
ah, good point. updating PR now